packages/shopping/src/__tests__/e2e/integration/profile.spec.js

```diff
@@ -0,0 +1,45 @@
+context('Profile', () => {
+  before(() => cy.visit('http://localhost:3000/shoppy.html'));
+
+  it('should update user profile', () => {
+    cy.get('#logIn')
+      .click()
+      .then(() => cy.get('#logInButton').click());
+
+    cy.get('#user')
+      .click()
+      .then(() => cy.get('.card-title').contains('John Doe'));
+
+    cy.get('input[name="profileFirstName"]')
+      .clear()
+      .type('Loopback')
+      .then(() =>
+        cy
+          .get('#update-profile')
+          .click()
+          .then(() => cy.get('#user').contains('Loopback Doe')),
```

Comment:
I'm not sure why this test is failing, but it might be a timing issue. Can you confirm if it works properly for you?

<img width="1194" alt="Screen Shot 2020-10-21 at 9 27 03 AM" src="https://user-images.githubusercontent.com/42985749/96726052-95cdd100-137f-11eb-8abd-04667278ac70.png">

Reply:
Probably same issue will add `.should('be.visible')` to ensure it always wait for the item to display